src/meta/src/hummock/manager/mod.rs

```diff
@@ -493,3 +501,68 @@ async fn write_exclusive_cluster_id(
         }
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct TableWriteThroughputStatistic {
+    pub throughput: u64,
+    pub timestamp: i64,
+}
+
+#[derive(Debug, Clone)]
+pub struct TableWriteThroughputStatisticManager {
+    table_throughput: HashMap<u32, Vec<TableWriteThroughputStatistic>>,
+    max_statistic_expired_time: i64,
```

Comment:
minor: it is better to put the unit of the timestamp in the name: `max_statistic_expired_secs`